src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
+                            media_image__150
+                            u-centered-on-mobile
+                            qa-resource-img"
```

Comment:
Another `media_image` could be added to the page potentially resulting in tests that incorrectly fail or, worse, incorrectly pass. My understanding with test selectors is that there shouldn't ever be the possibility of an ambiguous lookup. Now if the modules design is coming up with are hammered out and this will never ever have another `media_image` (on the page, in heros, status banners, etc.) then using `media_image` is probably fine, in the same vein as https://github.com/cfpb/cfgov-refresh/pull/915#discussion_r37442679

@sebworks brought up moving js- and qa- hooks to `data-js-hook` and `data-qa-hook` attributes that may make instances like this have a better separation of CSS and JS/QA uses.